sql/item_basic.sql

```diff
@@ -5904,8 +5904,8 @@ INSERT INTO `item_basic` VALUES (6268,0,'komanezumi','komanezumi',99,1548,63,0,0
 INSERT INTO `item_basic` VALUES (6269,0,'eminent_quiver','eminent_quiver',12,30292,99,0,0);
 INSERT INTO `item_basic` VALUES (6270,0,'eminent_bolt_quiver','em._bolt_quiver',12,30292,99,0,0);
 INSERT INTO `item_basic` VALUES (6271,0,'eminent_bullet_pouch','em._bul._pouch',12,30292,99,0,0);
-INSERT INTO `item_basic` VALUES (6272,0,'fried_popoto','fried_pototo',12,1548,54,0,0);
-INSERT INTO `item_basic` VALUES (6273,0,'fried_popoto_+1','fried_pototo_+1',12,1548,54,0,0);
+INSERT INTO `item_basic` VALUES (6272,0,'fried_pototo','fried_popoto',12,1548,54,0,0);
```

Comment:
Item 6272 is "Fried Popoto", item 6273 is "Fried Popoto +1" and the scripts have the proper names already (fried_popoto.lua and fried_popoto_+1.lua).

Reply:
strangely enough, the long name is still fried popoto, the sort name got changed to fried pototo, if i changed the names of the script ill check it and fix it
https://www.bg-wiki.com/bg/Fried_Popoto
https://www.bg-wiki.com/bg/Fried_Popoto_%2B1

Reply:
Hmm, that item seems to be a unique mess, maybe someone else can chime in.

![image](https://user-images.githubusercontent.com/17911103/37725341-2723c232-2d09-11e8-922b-f904e621195a.png)

![image](https://user-images.githubusercontent.com/17911103/37725357-2fe34730-2d09-11e8-883a-8a6380cce215.png)

Reply:
Guessing that was a mistake on SE's part. It's been "popoto" for years and I see the multi name still has that. I vote we leave "popoto" alone for now and see if another update corrects it.

Reply:
Looks like a case of bad english translation, in japanese it's listed as popoto, and that's what potatoes have been called from the start so lets go with that.

Reply:
ill revert both to popoto, and the scripts